app/views/incomes/yourIncomeCalculation.scala.html

```diff
@@ -124,6 +116,14 @@ <h2 class="heading-medium">@Messages("tai.income.calculation.heading",s"${Dates.
                                     </tr>
                                 }
                             </tbody>
+                            <tfoot>
```

Comment:
This change isn't needed. <tfoot> before <tbody> is valid and can sometimes be an improvement